src/MentorBot.Core/Abstract/Repositories/IQuestionRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) 2018. Licensed under the MIT License. See https://www.opensource.org/licenses/mit-license.php for full license information.
+
+using System.Threading.Tasks;
+
+using MentorBot.Data.Models;
+
+namespace MentorBot.Core.Abstract.Repositories
+{
+    /// <summary>
+    /// Repository for accessing data related to <see cref="Question"/>.
+    /// </summary>
+    public interface IQuestionRepository

```

Comment:
Please inherit from IDbRepository<Question>.
I don't see the need from this interface from at this point.
You can inject IDbRepository<Question> directly into the business layer for now, since there is no need from override.

Reply:
I will leave inheriting from `DbRepository`, since it has to pass the context. I will remove `IDbRepository` since as you said there is no need of this interface.